Tests/ApolloCodegenTests/CodeGeneration/FileGenerators/OperationManifestFileGeneratorTests.swift

```diff
@@ -71,7 +71,7 @@ class OperationManifestFileGeneratorTests: XCTestCase {
 
```

Comment:
I think we should also add tests for to ensure that we never get a doubled `.json.json` extension for both relative and absolute paths.